api/src/main/java/org/pmiops/workbench/db/model/DataSet.java

```diff
@@ -32,7 +32,7 @@
   private List<Long> conceptSetId;
   private List<Long> cohortSetId;
   private List<DataSetValues> values;
-  private short prePackagedConceptSet;
+  private Short prePackagedConceptSet;
```

Comment:
Why not store a `PrePackagedConceptSetEnum` here, and do the conversion on assignment?

Alternatively, you could use an `Optional<Short>` instead of just a boxed short. That would make things more obvious for the callers. Depends on whether 0 has special meaning in this enum that matches a notion of absense.

Reply:
0 has a meaning in the enum ('None' which results in none of the concept set selected on view) hence just keeping it Short solves the issue